src/components/BrewerySearch.js

```diff
@@ -15,21 +17,28 @@ export default class BrewerySearch extends Component {
 
   onSearchClick = e => {
     const { search } = this.state
-    const { handleSearch } = this.props
-    if (search) {
-      handleSearch(search)
+    let searchTrim = search.trim();
+    if (searchTrim.length > 1) {
```

Comment:
1)`searchTerm` here should be a **const** as it's value is not changed inside the function.

2)`searchTrim.length > 1` is kind of verbose. Instead `if (searchTrim)` is enough as the trim removes any empty space regardless of whether the user's search value is 
'`                                      `' or  '  ', changing the input to ''. This would make the input have a length of 0, and in an if statement 0 is falsy, but all other numbers are truthy.